src/Roslyn.Suppressions/Roslyn.Suppressions/SettingsFile/RoslynSettingsFileStorage.cs

```diff
@@ -31,7 +31,10 @@ namespace SonarLint.VisualStudio.Roslyn.Suppressions.SettingsFile
 {
     internal interface IRoslynSettingsFileStorage
     {
-        void Update(RoslynSettings settings);
+        /// <summary>
+        /// Updates the Roslyn settings file on disc for the specified solution
+        /// </summary>
+        void Update(RoslynSettings settings, string solutionNameWithoutExtension);
```

Comment:
We now need the simple solution name so we can save the file in the correct place.

I considered adding the solution name to the `RoslynSettings` object, but I felt it didn't belong there so I didn't.